### DISCUSSION TITLE
There is some way to not overflow the completion list?

### DISCUSSION THREAD

**Discussion by Zeioth** [hrsh7th/nvim-cmp#1587] [Q&A]
The behabior I'm looking for is:

* If you are on the top and press UP don't jump to the bottom.
* If you are on the bottom and press DOWM don't jump to the top.

Instead, I'd like to move to the top/bottom using HOME/END keys. How could I achieve this? Thank you for creating such a great plugin.

**Comment by Shougo** [comment]
Related https://github.com/hrsh7th/nvim-cmp/discussions/1582
